language/stdlib/modules/libra_system.mvir

```diff
@@ -17,6 +17,12 @@ module LibraSystem {
         new_validator_set: Vector.T<Self.ValidatorInfo>,
     }
 
+    struct NewBlockEvent {
```

Comment:
We can easily put additional data in here (e.g., the proposer address.) if it would be useful to clients